com.github.iwalton3.jellyfin-media-player.json

```
@@ -0,0 +1,275 @@
+{
+  "app-id": "com.github.iwalton3.jellyfin-media-player",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "base": "io.qt.qtwebengine.BaseApp",
+  "base-version": "5.15",
+  "command": "jellyfinmediaplayer",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--device=all",
+    "--talk-name=org.freedesktop.PowerManagement",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--system-talk-name=org.freedesktop.login1"
+  ],
+  "build-options": {
+    "strip": true
+  },
```

```suggestion
  "build-options": {
    "strip": true
  },
```

This is the default.